Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -584,7 +584,7 @@ void MainWindow::ConnectStack()
 
   setCentralWidget(m_stack);
 
-  setDockOptions(DockOption::AllowNestedDocks);
+  setDockOptions(DockOption::AllowNestedDocks | DockOption::AllowTabbedDocks);
```

Comment:
You might want to set DockOption::AnimatedDocks too, as the default is `AnimatedDocks | AllowTabbedDocks`

Reply:
That option was causing crashes when I dragged a widget around.